kempner_hpc_handbook/getting_started/accessing_and_navigating_the_cluster.md

```diff
@@ -16,7 +18,11 @@ To set up a VPN connection, follow these steps:
     - **2FA code**: your 2FA code (Set up 2FA [here](https://docs.rc.fas.harvard.edu/kb/openauth/))
 
 ```{note}
-For detailed instructions and additional information, please read more about the VPN at [VPN setup guide](https://docs.rc.fas.harvard.edu/kb/vpn-setup/).
+For detailed instructions and additional information, please read more about the VPN at [VPN setup guide](https://docs.rc.fas.harvard.edu/kb/vpn-setup/) on FASRC Documentation.
+```
+
+```{warning}
+Please avoid logging in directly to specific login nodes, as this can disrupt the load balancing on the nodes and may exert undue pressure on a particular login node (e.g., `holylogin01` through `holylogin04`). An exception exists for VSCode, which requires rotation between login nodes. Please refer to the [VSCode Section](../development_and_runtime_envs/using_vscode_for_remote_development.md) for more information.
```

Comment:
Please use labels for cross referencing. 
`[VSCode Section](../development_and_runtime_envs/using_vscode_for_remote_development.md)` --> `[VSCode ](development_and_runtime_envs:using_vscode_for_remote_development)`
Please see more details here: https://jupyterbook.org/en/stable/content/references.html#reference-section-labels